packages/daisyui/src/components/modal.css

```diff
@@ -95,6 +95,9 @@
     box-shadow: oklch(0% 0 0/ 0.25) 0px 25px 50px -12px;
     overflow-y: auto;
     overscroll-behavior: contain;
+    @media (prefers-reduced-motion: reduce) {
```

Comment:
Change this to use the general pattern of only applying the motion transition when `prefers-reduced-motion: no-preference`. The non-motion transition should be applied always.